spec/clinic.v1.yaml

```diff
@@ -1812,6 +1812,25 @@ paths:
           required: true
       x-internal: true
     parameters: []
+  '/v1/clinics/{clinicId}/service_accounts':
+    parameters:
+      - $ref: '#/components/parameters/clinicId'
+    post:
+      summary: Add Service Account
+      operationId: AddServiceAccount
+      responses:
+        '200':
```

Comment:
I don't think 200 is the only possible response..?

...or is this a generated file? I wonder because of all the stoplight ids.

Reply:
This is not a generated file, but it was edited with the Stoplight editor. 200 is the only valid response and given this is for internal use only (will be called from ORCA or manually using CURL) I think it's good enough. Does documenting 401 or 500 provide any benefit?

Reply:
> Does documenting 401 or 500 provide any benefit?
Not really, but it's done in most of the API endpoints' documentation, so I thought it was something the team did.